src/sentry/static/sentry/app/components/smartSearchBar/index.jsx

```diff
@@ -446,6 +452,38 @@ class SmartSearchBar extends React.Component {
     ];
   };
 
+  getReleases = _.debounce(
+    async (tag, query) => {
+      const releasePromise = this.fetchReleases(query);
+
+      const tags = this.getPredefinedTagValues(tag, query);
+      const tagValues = tags.map(v => ({
+        ...v,
+        type: 'first-release',
+      }));
+
+      const releases = await releasePromise;
```

Comment:
What catches the errors when this request fails?

Reply:
Good catch. I've added try/catch at the fetch function inside the component.